src/subscription_manager/managercli.py

```diff
@@ -2616,7 +2617,20 @@ def _do_command(self):
         else:
             result = 1
 
-        print(_("Overall Status: %s\n") % service_status['status'])
+        ca_message = ""
+        has_cert = (_(
+                "Content Access Mode is set to Organization/Environment Access. This host has access to content, regardless of subscription status.\n"))
+
+        certs = self.entitlement_dir.list_with_content_access()
+        ca_certs = [cert for cert in certs if cert.entitlement_type == CONTENT_ACCESS_CERT_TYPE]
+        if ca_certs:
+            ca_message = has_cert
+        else:
+            owner = self.cp.getOwner(self.identity.uuid)
```

Comment:
We should wrap this in a Try .. except and log at the debug level that we're unable to check for the content access mode due to the exception.